djangocms_versioning/admin.py

```diff
@@ -31,17 +31,20 @@
 
 
 class VersioningChangeListMixin:
+    """Mixin used for ChangeList classes of content models."""
 
     def get_queryset(self, request):
+        """Limit the content model queryset to latest versions only."""
         queryset = super().get_queryset(request)
         versionable = versionables.for_content(queryset.model)
         return queryset.filter(pk__in=versionable.distinct_groupers())
 
 
-def versioning_change_list_factory(ChangeList):
+def versioning_change_list_factory(BaseChangeListClass):
```

Comment:
Think if you add `class` to the name, then convert it to snake case (`base_change_list_cls`?)